src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+        System.out.println("Hello I'm Duke\nWhat can I do for you?");
+        Scanner scan = new Scanner(System.in);
+        
+        while (true) {
+            String textInput = scan.nextLine();
+
+            if (textInput.equalsIgnoreCase("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                scan.close();
+                return;
+            }
+
+            if (textInput.equalsIgnoreCase("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskList.size(); i++) {
+                    System.out.println(i + 1 + ". " + taskList.get(i).toString());
+                }
+                continue;
+            }
+
+            if (textInput.length() >= 8 &&
+                    textInput.substring(0, 6).equalsIgnoreCase("delete")) {
+                int i = Integer.parseInt(textInput.substring(7));
+                Task t = taskList.get(i - 1);
+                taskList.remove(i - 1);
+                String output = String.format("Got it. I've removed this task:\n%s\nNow you have %d tasks in the list", t.toString(), taskList.size());
+                System.out.println(output);
+                continue;
+            }
+
+            if (textInput.length() >= 6 && 
+                    textInput.substring(0, 4).equalsIgnoreCase("mark")) {
+                int i = Integer.parseInt(textInput.substring(5));
+                Task currTask = taskList.get(i - 1);
+                currTask.markDone();
+                System.out.println("Nice! I've marked this task as done\n" + currTask.toString());
+                continue;
+            }
+
+            if (textInput.length() >= 8 &&
+                    textInput.substring(0, 6).equalsIgnoreCase("unmark")) {
+                int i = Integer.parseInt(textInput.substring(7));
+                Task currTask = taskList.get(i - 1);
+                currTask.markUndone();
+                System.out.println("OK, I've marked this task as not done yet:\n" + currTask.toString());
+                continue;
+            }
+
+            if (textInput.length() >= 4 &&
+                    textInput.substring(0, 4).equalsIgnoreCase("todo")) {
+                try {
+                    String[] parts = textInput.split(" ", 2);
+                    if (parts.length == 1 || parts[1] == "") {
+                        throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+                    }
+                    Task t = new Task.Todo(textInput.substring(5));
+                    taskList.add(t);
+                    String output = String.format("Got it. I've added this task:\n%s\nNow you have %d tasks in the list", t.toString(), taskList.size());
```

Comment:
This output line seems very long, is it possible to split them into 2 lines?
A similar issue appears in line 32, 78, 88.